package.json

```diff
@@ -61,7 +62,7 @@
     "@types/lodash": "4.14.63",
     "@types/mocha": "2.2.40",
     "@types/mz": "0.0.31",
-    "@types/node": "7.0.14",
+    "@types/node": "6.0.70",
```

Comment:
Because we the minimum Node version we can rely on is Node 6 (LTS). If we used Node 7, we would be able to use `require('url').URL` instead of the polyfill